packages/Tabs/src/Tabs.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from "prop-types";
+import TabsContext from "./TabsContext";
+import Panel from "./components/Panel/Panel";
+import Panels from "./components/Panels/Panels";
+import Tab from "./components/Tab/Tab";
+import List from "./components/List/List";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  isDisabled: PropTypes.node,
+};
+
+const defaultProps = {
+  isDisabled: false,
+};
+
+const Tabs = props => {
+  const [activeIndex, setActiveIndex] = React.useState(null);
+  const [numberOfTabs, setNumberOfTabs] = React.useState(null);
+  let tabListRef = null;
+
+  const { isDisabled } = props;
+
+  const setTabListRef = ref => {
+    tabListRef = ref;
+  };
+
+  const handleTabClick = (event, index) => {
+    event.preventDefault();
+    if (activeIndex !== index) {
+      setActiveIndex(index);
+    }
+  };
+
+  const handleKeyDown = index => {
+    let tabIndex = index;
+    const totalTabs = numberOfTabs;
+
+    if (index < 0) {
+      tabIndex = numberOfTabs - 1;
+    } else if (index >= totalTabs) {
+      tabIndex = 0;
+    }
+
+    tabListRef.querySelectorAll(".tab")[tabIndex].focus();
```

Comment:
maybe `[role="tab"]` ?

Reply:
To be flexible, it may not always be a role tab. For example, if the tabs are used for navigation purposes then the consumer will pass in a role manually, so the role is not a unique value